playbooks/roles/bats/defaults/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
+bats_git_dir: "/root/bats"
+bats_forklift_dir: "/root/forklift"
+bats_update_forklift: "yes"
+bats_tests: "fb-content-katello.bats fb-capsule.bats fb-finish-katello.bats"
```

Comment:
Personally I prefer the explicitness of filenames over the shorter named ```katello-bats``` versions. To me it's more obvious and allows for grepping where a file is used.